src/main/java/com/aspose/imaging/cloud/sdk/invoker/internal/ApiInvoker.java

```diff
@@ -78,7 +78,7 @@
      */
     public ApiInvoker(IRequestHandler[] requestHandlers)
     {
-        String sdkVersion = "18.4";
+        String sdkVersion = "18.6";
```

Comment:
@MaxCrank I think it makes sense to move the version to the common constant.

Reply:
@roxboris you're right, fixed.